### ISSUE TITLE
Disable ambience not saved and not exported.

### ISSUE THREAD

**Issue by Hohenheimsenberg** [UnderwaterApps/overlap2d#60]
I have ambience disabled in my scene, even if I save it and open the scene again, it gets enabled. 
Also when I export the scene and load my game, ambience is enabled too.

**Comment by azakhary** [comment]
The disable/enable ambience feature are currently for lookup only, so you can work with it without lights staying in the way. If you want to have no ambience in your actual scene, you can either set it to white from editor, or you can use simple Stage instead of Overlap2DStage.

**Comment by Hohenheimsenberg** [comment]
I checked and the exported scene has ambient color set to white
``` 
"ambientColor":[1,1,1,1]
```
I have to set it in code to make it work:
```
Color clr = new Color(1,1,1,1);
essentials.rayHandler.setAmbientLight(clr);
```

I'm wondering is my workflow wrong?
```
rm = new ResourceManager();
rm.loadProjectVO();
rm.initScene("scene");
myStage = new MyStage(rm);
...
```

And in MyStage
```
public class MyStage extends Overlap2DStage{
    public MyStage(ResourceManager rm) {
       super(new ExtendViewport(width, height));
       sceneLoader = new SceneLoader(rm);
       sceneLoader.setResolution(rm.currentResolution.name);
       sceneLoader.loadScene(GardenScene.SCENE_NAME);		
       addActor(sceneLoader.getRoot());        
...
}
```

**Comment by Hohenheimsenberg** [comment]
Found my mistake, instead of just creating a new SceneLoader had to use initSceneLoader().
But now I have another issue, when resizing the desktop window I get a flicker, first frame: new size, second frame: new size plus last size, third frame: new size, and so on.

Im using ExtendViewport, one frame right and next wrong. Here I resized it down, so the right image is the smaller one.
![image](https://cloud.githubusercontent.com/assets/4349541/7188649/a31c7326-e436-11e4-8500-9142f8f4910e.png)
